pyproject.toml

```diff
@@ -209,14 +209,25 @@ select = [
     "SIM", # flake8-simplify
     "TID251", # banned-api
     "T201", # print
+    "D" # pydocstyle
 ]
 ignore = [
     "UP007", # non-pep604-annotation
     "E501", # line-too-long
     "B024", # abstract-base-class-without-abstract-method
     "B009", # get-attr-with-constant
     "B010", # set-attr-with-constant
+    "D100", # undocumented-public-module
+    "D101", # undocumented-public-class
+    "D102", # undocumented-public-method
+    "D103", # undocumented-public-function
+    "D104", # undocumented-public-package
+    "D105", # undocumented-magic-method
+    "D106", # undocumented-public-nested-class
+    "D107", # undocumented-public-init
```

Comment:
Out of curiosity were you planning on un-ignoring any of these later/incrementally?

Reply:
Yeah, eventually we should run ruff on changed files only and remove the ignored rules.